cluster/postgres-operator-dependencies.yaml

```diff
@@ -0,0 +1,175 @@
+SenzaComponents:
```

Comment:
Why do we need this stack?

It seems like it's quite out of scope for our base Kubernetes infrastructure to also provision extra *non-cluster* related stacks.

We already have the problem of manually updating the existing etcd stack when taupage is outdated etc. Adding another etcd is not making it simpler.